test/data/test_utils.py

```diff
@@ -48,3 +49,12 @@ def test_text_nomalize_function(self):
                 ref_lines.append(line)
 
         self.assertEqual(ref_lines, test_lines)
+
+    def test_version_parsing(self):
+        import torch
+        # Assign a torch version arbitrarily
+        version = (1, 10, 1)
+        major, minor, patch = version
+        torch.__version__ = ".".join(map(str, version))
```

Comment:
This effect persists after the test is done. Either perform the clean up or change the signature of `get_torch_version` to accept the value to parse.

Reply:
I guess this begs the question: where is this function used? I did a quick scan of the repo and couldn't find it anywhere. Is it an artifact of legacy API?

Reply:
Indeed I got the same question and looks like the it's not really torchtext's responsibility or interest to provide this function.

@parmeet @abhinavarora what do you think?

Reply:
Yes, We are in process of cleaning up stale-code and this is potential candidate. I agree with @mthrok that it's not in torchtext's interest to maintain this function.

Reply:
If it's on the chopping block, shall I repurpose this PR to remove it? It's technically part of the public API, but I can at least deprecate it if desired.

Reply:
I think it makes sense to repurpose this PR for deprecation, but let's hear from @parmeet .